pom.xml

```diff
@@ -31,7 +31,7 @@
     <artifactId>gravitee-am-parent</artifactId>
     <packaging>pom</packaging>
 
-    <version>4.0.17-SNAPSHOT</version>
+    <version>4.0.18-SNAPSHOT</version>
```

Comment:
why is this version updated?

Reply:
Sorry, this was a mistake in the PR commits. Fixed.